namespaces/live-1.cloud-platform.service.justice.gov.uk/check-my-diary-dev/01-rbac.yaml

```diff
@@ -0,0 +1,13 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: check-my-diary-dev-admin
+  namespace: check-my-diary-dev
+subjects:
+  - kind: Group
+    name: "github:check-mny-diary"
```

Comment:
watch the typo :)

also add admin permissions for dso in sheffield

Reply:
fixed the typo and added DSO to admins